src/components/Conclusion.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div class="bg-washed-blue">
```

Comment:
The `bg-washed-blue` class is at the `App` level. Can we update this to `conclusion`?